packages/flutter_tools/templates/module/android/gradle/build.gradle.tmpl

```diff
@@ -18,6 +20,16 @@ allprojects {
     }
 }
 
-task clean(type: Delete) {
```

Comment:
Why are these two lines being removed?

Reply:
the clean task is already provided by the `com.android.library` plugin applied right below this change